pkg/probe/http/http.go

```diff
@@ -134,7 +134,7 @@ func DoHTTPProbe(url *url.URL, headers http.Header, client GetHTTPInterface) (pr
 		klog.V(4).Infof("Probe succeeded for %s, Response: %v", url.String(), *res)
 		return probe.Success, body, nil
 	}
-	klog.V(4).Infof("Probe failed for %s with request headers %v, response body: %v", url.String(), headers, body)
+	klog.V(4).Infof("Probe failed for %s with request headers %v, response body: %v", url.String(), headers, nil)
```

Comment:
What's the reason for continuing to include formatting for `response body: %v` if it'll always be `nil`?

Reply:
That was a mistake on my part. I misread the ask for eliding the body. I'll remove that commit and push a new one shortly.